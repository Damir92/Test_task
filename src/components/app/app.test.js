import React from 'react';
import {shallow} from 'enzyme';

import App from './app';

describe(`App component`, () => {
  it(`App snapshot`, () => {
    const tree = shallow(
        <App />
    );

    expect(tree).toMatchSnapshot();
  });
});
