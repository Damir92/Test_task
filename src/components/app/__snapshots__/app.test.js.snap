// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component App snapshot 1`] = `
<BrowserRouter>
  <Switch>
    <Route
      component={[Function]}
      exact={true}
      path="/"
    />
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(Users)",
          "type": [Function],
        }
      }
      exact={true}
      path="/users"
    />
  </Switch>
</BrowserRouter>
`;
